packages/core-transactions/src/handlers/two/delegate-resignation.ts

```diff
@@ -74,30 +82,12 @@ export class DelegateResignationTransactionHandler extends TransactionHandler {
         emitter.dispatch(Enums.DelegateEvent.Resigned, transaction.data);
     }
 
-    public async canEnterTransactionPool(
-        data: Interfaces.ITransactionData,
-        pool: Contracts.TransactionPool.Connection,
-        processor: Contracts.TransactionPool.Processor,
-    ): Promise<boolean> {
-        // TODO: ioc
-        if (await this.typeFromSenderAlreadyInPool(data, pool, processor)) {
-            // @ts-ignore
-            const wallet: Contracts.State.Wallet = pool.poolWalletRepository.findByPublicKey(data.senderPublicKey);
-            processor.pushError(
-                data,
-                "ERR_PENDING",
-                `Delegate resignation for "${wallet.getAttribute("delegate.username")}" already in the pool`,
-            );
-            return false;
-        }
-
-        return true;
-    }
-
     public async applyToSender(
         transaction: Interfaces.ITransaction,
         customWalletRepository?: Contracts.State.WalletRepository,
     ): Promise<void> {
+        this.app.log.notice(`${this.walletRepository.allByUsername().length} delegates before resignation`);
```

Comment:
Inject the logger instead of using `this.app.log`. The `log` getter will be removed soon, it was only added initially for migration purposes.

Reply:
Let me remove that, I left it there by accident while debugging